### DISCUSSION TITLE
using loadable breaks the dependancy in Jotai Devtools

### DISCUSSION THREAD

**Discussion by yassuge** [pmndrs/jotai#2125] [Q&A]
Hi, I m using loadable, to wrap an async atom. However, any atom does inside loadable, is not showing up  the loadable in the dependents section in devtools. Is this a known issue?

Many thanks

**Comment by dai-shi** [comment]
@yassuge Would you be able to create a reproduction with https://csb.jotai.org ?

@arjunvegda Is it because of `debugPrivate` ?

**Reply by arjunvegda** [reply]
Yeah, I'm able to reproduce it ([Here is a CSB](https://codesandbox.io/s/jotai-discussion-2125-6h6d3j?file=/src/App.tsx)), the async atom doesn't show up in dependents section of loadable atom for some reason 🤔 I'll look into it

**Reply by yassuge** [reply]
I reproduced here as well in case:
https://codesandbox.io/s/headless-star-wq238c?file=/src/App.tsx

**Reply by arjunvegda** [reply]
False alarm, you're right @dai-shi, It is indeed due to `debugPrivate`. The async atom user passes is wrapped by a derived atom that [we mark as private](https://github.com/pmndrs/jotai/blob/main/src/vanilla/utils/loadable.ts#L60), which causes it to not show up in the dev tools. 

To see all the private atoms, users would need to set `shouldShowPrivateAtoms` to `true`
```diff
    <DevTools
      isInitialOpen
+    options={{
+      shouldShowPrivateAtoms: true
+    }}
    />
```

**Reply by yassuge** [reply]
Thanks a lot for your help @arjunvegda and @dai-shi , this solves it!
